code/modules/buildmode/submodes/fill.dm

```diff
@@ -1,6 +1,9 @@
+#ifndef FILL_WARNING_MIN
+#define FILL_WARNING_MIN 150
```

Comment:
Why the `#ifndef` ?

Reply:
Holdover from C++, I'm guessing it's unnecessary

Reply:
Mhm, just remove that, the operation is redundant and we don't expect `FILL_WARNING_MIN` to be defined elsewhere.
If it is, the linter will pick up the duplicate definitions and throw a warning.